src/app/common/services/bootstrap/bootstrap.service.ts

```diff
@@ -28,6 +63,12 @@ export class BootstrapService {
 
   addConfig(gw: Gateway) {
     // Boostrap
+    this.content.export_config.mqtt.channel = gw.metadata.exportChannelID;
+    this.content.export_config.mqtt.username = gw.id;
+    this.content.export_config.routes[0].mqtt_topic = 'channels/' + gw.metadata.exportChannelID + '/messages';
+    this.content.export_config.routes[1].mqtt_topic = 'channels/' + gw.metadata.exportChannelID + '/messages';
+    this.content.export_config.mqtt.password = gw.key;
+
```

Comment:
Is there a particular reason to mix CamelCase and snake_case?

Reply:
This follows style of other bootstraping variables (like `log_level`, `http_port`). 
Anyway, this depends on the backend that requires this variables.

Reply:
Maybe we could change this in backend, puting some marshalling hints in the go code. Anyway, for now, I understand the need for the case mixing.